assets/src/dashboard/components/inlineInputForm/index.js

```diff
@@ -35,6 +35,7 @@ const InlineInputForm = ({
   onEditCancel,
   onEditComplete,
   value,
+  ...rest
```

Comment:
make sure we can pass in things like `placeholder` as well without having to keep adding props here that don't affect what the component needs to do.

Reply:
This is where I miss TypeScript a bit because we could confine the props to HTML Input. There are some cases where we could pass too much. I almost want to air on the side of being explicit about `placeholder`, `etc`.

Reply:
sure thing! I'll update this. Honestly, not much else is going to come up here and if it does it's good to know about it.